### PR TITLE
feat: Update General tab in repository settings IO-201 CY-6727

### PR DESCRIPTION
**Pull request by prcr**
Updates the pages that include screenshots or that mention the General tab in the repository settings.

### :eyes: Live preview
- https://feat-update-settings-general-io-201--docs-codacy.netlify.app/getting-started/adding-a-codacy-badge/
- https://feat-update-settings-general-io-201--docs-codacy.netlify.app/repositories-configure/using-submodules/
- https://feat-update-settings-general-io-201--docs-codacy.netlify.app/repositories-configure/removing-your-repository/
- https://feat-update-settings-general-io-201--docs-codacy.netlify.app/related-tools/local-analysis/running-aligncheck/
- https://feat-update-settings-general-io-201--docs-codacy.netlify.app/faq/repositories/i-renamed-my-repository-on-the-git-provider/
- https://feat-update-settings-general-io-201--docs-codacy.netlify.app/faq/troubleshooting/we-no-longer-have-access-to-this-repository/

### CODE REVIEW

**Comment by prcr** [comment]
I'm closing this pull request since the work is stalled.

**Comment by nicklem** [comment]
I'm reopening this one, since things started moving again on the [relevant epic](https://codacy.atlassian.net/browse/IO-467).

File: docs/faq/repositories/i-renamed-my-repository-on-the-git-provider.md [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 If you changed the name or URL of your repository on your Git provider, you can update the name and URL of the repository on Codacy to point to the new location. This ensures that you won't lose historical data about your repository on Codacy.
 
-To rename your repository on Codacy, open the page **Settings** and click the button **Update name**.
+To rename your repository on Codacy, open the page **Settings** and click the button **Update repository**.
```
Comment (nicklem) [inline docs/faq/repositories/i-renamed-my-repository-on-the-git-provider.md:5]: Here the label of the button changes, as well as its capabilities. We used to have two buttons here.

I'm opting not to expand on this, given that the context of the page.

**Comment by prcr** [comment]
I can't approve my own pull request :sweat_smile: but the changes look good to me. :ok_hand:

**Review by nicklem — APPROVED** [review]
Approving on behalf of @prcr :shipit:

**Pull request merged**